src/Parts/WelcomeScreen.php

```diff
@@ -8,7 +8,7 @@ class WelcomeScreen
 {
     public ?string $description;
     /**
-     * @var \Ragnarok\Fenrir\Parts\WelcomeScreenChannel[]
+     * @var WelcomeScreenChannel[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)